src/stylesheets/components/_user_card.scss

```diff
@@ -12,10 +12,11 @@
   border-radius: 5px;
   box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.1);
   cursor: pointer;
-  transition: bottom .2s ease-in-out;
+  transition: all .2s ease-in-out;
 
   &:hover {
     bottom: 10px;
+    box-shadow: 0px 15px 40px rgba(0, 0, 0, 0.2);
```

Comment:
another case... maybe predefine shadow?

Reply:
Yeah, that's a good point. I'm gonna create `$shadow` and $shadow-hover` variables.